devel/kubecolor/Portfile

```
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/kubecolor/kubecolor v0.5.0
```

The format of `go.setup` is:

```
go.setup            domain/author/project version [tag_prefix] [tag_suffix]
```

Refer to https://guide.macports.org/#reference.portgroup.golang.setup

So this should look like:

```suggestion
go.setup            github.com/kubecolor/kubecolor 0.5.0 v
```

---

Thank you for the review and suggestions! This is my first contribution to MacPorts, so it might not fully follow the conventions.